partner_second_lastname/tests/test_name.py

```diff
@@ -130,25 +141,29 @@ def test_firstname_last(self):
         }
 
     def test_firstname_only(self):
```

Comment:
this method should be called test_lastname_only isn't? does test_firstname_only is tested somewhere else?